tests/h/views/api/groups_test.py

```diff
@@ -184,6 +186,78 @@ def pyramid_request(self, pyramid_request, factories):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('group_service',
+                         'user_service',
+                         'request_auth_client',
+                         'validate_auth_client_authority')
+class TestAddMember(object):
+
+    def test_it_adds_current_user_to_group(self,
+                                           factories,
+                                           pyramid_request,
+                                           group_service,
+                                           user_service):
+        group = factories.Group()
```

Comment:
Can you please instantiate both of these User and Group objects with an explicit authority, just so it's visible that authority matters?